tests/unit_tests/migrations/viz/pivot_table_v1_v2_test.py

```diff
@@ -14,122 +14,40 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-import json
+from typing import Any
 
 from superset.migrations.shared.migrate_viz import MigratePivotTable
-from tests.unit_tests.conftest import with_feature_flags
+from tests.unit_tests.migrations.viz.utils import migrate_and_assert
 
-SOURCE_FORM_DATA = {
-    "adhoc_filters": [],
+SOURCE_FORM_DATA: dict[str, Any] = {
     "any_other_key": "untouched",
     "columns": ["state"],
     "combine_metric": True,
-    "granularity_sqla": "ds",
```

Comment:
Moved the time parts to a separate test file called `time_related_fields_test.py`